packages/ignite-integration-tests/test/ignite-new/newMin.test.js

```diff
@@ -1,79 +0,0 @@
-const test = require('ava')
```

Comment:
Oh sorry... ya i should have said something.

This is going to move over to `ignite-ir-app`.   

It'll come back strong, faster, and more glorious than before.

I removed it because `--max` now comes with husky support (which is git hooks).  And when you have .git inside .git, things don't work well.   We might have to move the running directory into a `/tmp` directory.  

I didn't want to deal with that change right now, so I panicked and hit the delete button instead.